Userland/Libraries/LibVideo/Color/ColorPrimaries.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Matrix3x3.h>
+#include <LibVideo/Color/CodingIndependentCodePoints.h>
+#include <LibVideo/DecoderError.h>
+
+namespace Video {
+
+void convert_rgb_to_xyz(float& red, float& green, float& blue, ColorPrimaries primaries);
+
+void convert_xyz_to_rgb(float& red, float& green, float& blue, ColorPrimaries primaries);
```

Comment:
these two free functions are not used in `LibVideo: Implement video color space conversion`. Addtionally, they are quite non-idiomatic for Serenity, we tend to avoid reference params wherever possible. If this is needed in a later commit, it should be converted to take a FloatVec3 by reference, at least. it doesn't make much sense to list out the component separately and then create a float vec in the implementation.

Looking at it closer, it looks like it will do a copy of the floats to do the calculation anyway? so taking a floatVec and returning one is probably not that more expensive than what it's currently doing.

Reply:
Sure, I'll remove these, they were left over from a previous implementation, so they're not needed at least for now.

Certainly if this part of the library ever needs use elsewhere, we can remake this in a more useful signature, probably taking Gfx::Color as a parameter.